app/src/main/res/layout/fragment_team.xml

```diff
@@ -62,7 +62,7 @@
                 android:layout_height="wrap_content"
                 android:gravity="center"
                 android:padding="@dimen/padding_small"
-                android:text="Type"
+                android:text="@string/typename"
```

Comment:
@Avinash-Codes not sure if this solves the issue
in other words think this and following change are not necessary